### PR TITLE
fix regression for bert, we broke if multiple model outputs came from the same node

### PR DESCRIPTION
**Pull request by guschmue**
less than idea fix for now but some folks are blocked on it.

### CODE REVIEW

File: tf2onnx/tfonnx.py [hunk 1]
```diff
@@ -250,11 +250,18 @@ def arg_minmax_op(ctx, node, name, args):
...
 def reduce_op(ctx, node, name, args):
     axes_node = node.inputs[1]
-    input_rank = len(ctx.get_shape(node.input[0]))
     axes = axes_node.get_tensor_value()
     if np.isscalar(axes):
         axes = [axes]
-    axes = [val + input_rank if val < 0 else val for val in axes]
+    input_shape = ctx.get_shape(node.input[0])
+    if input_shape is None:
+        for val in axes:
```
Comment (pengwa) [inline tf2onnx/tfonnx.py]: maybe use any([val < 0 for val in axes])

File: tf2onnx/graph.py [hunk 2]
```diff
@@ -338,8 +338,20 @@ def __init__(self, nodes, output_shapes=None, dtypes=None, target=None, opset=No
...
         for o in self.outputs:
             n = self.get_node_by_output_in_current_graph(o)
+            if n in nodes_seen:
+                multi_output_nodes.add(n)
+            else:
+                nodes_seen.add(n)
+
+        for o in self.outputs:
+            n = self.get_node_by_output_in_current_graph(o)
+            # TODO: below doesn't work for nodes with multiple outputs. A work around, keep those intact.
```
Comment (pengwa) [inline tf2onnx/graph.py]: sorry for this regression. We can add the test just we did usually in the yaml file, could you please provide bert model and yaml file?

File: tf2onnx/tfonnx.py [hunk 3]
```diff
@@ -250,11 +250,18 @@ def arg_minmax_op(ctx, node, name, args):
...
+            if val < 0:
+                raise ValueError("reduce_op: can have negative axis because we don't know input rank")
```
Comment (pengwa) [inline tf2onnx/tfonnx.py]: do you intent to use "can**not** have negative axis"?

File: tf2onnx/graph.py (hunk 2 above)
Comment (guschmue) [reply tf2onnx/graph.py]: no worry, this is the only model I know that takes multiple outputs from the same node as model output. bert needs a little extra training before its ready to convert ... need to find a place to download it from and its large. Maybe I can add a unit test to simulate the thing.

File: tf2onnx/tfonnx.py (hunk 1 above)
Comment (guschmue) [reply tf2onnx/tfonnx.py]: sold

File: tf2onnx/tfonnx.py (hunk 3 above)
Comment (guschmue) [reply tf2onnx/tfonnx.py]: yep.

**Pull request merged**